specification/vmware/resource-manager/readme.md

```diff
@@ -239,10 +254,43 @@ suppressions:
     # https://azure.github.io/autorest/extensions/#x-ms-long-running-operation-options
     from: vmware.json
 
-  # just warnings
-  # - code: IgnoredPropertyNextToRef
-  # - code: LroLocationHeader
-  #   from: vmware.json
+  - code: XmsClientName
+    reason: false positives
+    from: vmware.json
+
+  - code: XmsClientNameParameter
+    reason: this warning is false postive
+    from: vmware.json
+
+  - code: docLinkLocale
+    reason: false positives
+    from: vmware.json
+
+  - code: PropertiesTypeObjectNoDefinition
+    reason: false positives
+    from: vmware.json
+
+  - code: LatestVersionOfCommonTypesMustBeUsed
+    reason: this warning is TODO
+    from: vmware.json
+
+  - code: AvoidAdditionalProperties
+    reasons: this is how TypeSpec models records
+    from: vmware.json
+
+  - code: XmsPageableForListCalls
+    reasons: This is a TypeSpec @singleton.
+    from: vmware.json
+    # where: WorkloadNetworks_Get
+
+  - code: DeleteResponseCodes
+    reasons: false positives
+    # delete is ArmResourceDeleteAsync<IscsiPath>;
+
+  - code: RequestBodyMustExistForPutPatch
+    reasons: false positives
+    # createOrUpdate is ArmResourceCreateOrUpdateAsync<IscsiPath>;
```

Comment:
>ArmResourceCreateOrUpdateAsync [](http://example.com/codeflow?start=24&length=30)

it seems like all the properties of IscisiPath are readonly, am I understanding it right? Then how does it have a create/update action?

Reply:
```
  @doc("CIDR Block for iSCSI path.")
  @visibility("create", "read")
  networkBlock: string;
```

Reply:
Will close. `networkBlock` is set during a create.